wcfsetup/setup/db/install.sql

```diff
@@ -1446,6 +1446,20 @@ CREATE TABLE wcf1_trophy(
 	KEY(categoryID)
 );
 
+DROP TABLE IF EXISTS wcf1_unfurl_url;
+CREATE TABLE wcf1_unfurl_url (
+	urlID INT(10) NOT NULL AUTO_INCREMENT PRIMARY KEY,
+	url TEXT NOT NULL,
+	urlHash VARCHAR(40) NOT NULL UNIQUE KEY (urlHash),
```

Comment:
You misunderstood the part regarding the UNIQUE KEY. It should go at the bottom (similar to `username` for `wcf1_user`).